src/auto_scheduler/feature.cc

```diff
@@ -1345,11 +1345,6 @@ void GetPerStoreFeaturesFromStates(const Array<State>& states, const SearchTask&
                           GetPerStoreFeaturesWorkerFunc(task, states[i], max_n_bufs,
                                                         &(*features)[i], &error_ct);
                         });
-
-  if (error_ct > 0) {
-    std::cerr << "Encountered " << error_ct
-              << " errors during feature extraction, which are safely ignored." << std::endl;
-  }
```

Comment:
Can we keep this message in a lower logging verbose level? If not (which I think might be an issue because DMLC logging system doesn't have DEBUG level), we may need to add a message at the end of the measurement process indicating the case that all states are failed.

Reply:
This kind of check will happen in other places (e.g., in the search policy).